docs/api.md

```diff
@@ -857,6 +858,10 @@ Example:
 nativefier <your-geolocation-enabled-website> --win32metadata '{"ProductName": "Your Product Name", "InternalName", "Your Internal Name", "FileDescription": "Your File Description"}'
 ```
 
+#### [disable-old-nativefier-warning]
+
+Disables warning when opening an app made with an old version of Nativefier. 
+
```

Comment:
Please document that it's insecure, and require an explicit argument forcing users to realize, and type themselves, that this is insecure. See https://github.com/jiahaog/nativefier/issues/556 :

> *"Should default to true, and be deactivable but with a very visible flag, e.g. `--deactivate-old-electron-warning yesiknowitisinsecure`."*